exes/ethercat/CMakeLists.txt

```diff
@@ -47,7 +47,7 @@ install(
   FILES
     systemd/ethercat.service
   DESTINATION
-    ${CMAKE_INSTALL_LIBDIR}/systemd/user/
+    /usr/lib/systemd/system/
```

Comment:
why does ${CMAKE_INSTALL_LIBDIR} not work?

Reply:
Sometimes it is the wrong thing see.  https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html 

And it kind of does not make sense. Why try to extract a specific location from variables.
This placement is not variable.

Reply:
I think you will break vcpkg usage of the framework. So I would like to investigate or test the vcpkg. This can also be postponed to a separate issue ticket.

Reply:
Let's postpone this until confirmed to be an issue. Can be built using BUILD_EXES=False for now.